tests/python/pants_test/rules/test_filedeps_integration.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class FiledepsIntegrationTest(PantsRunIntegrationTest):
+
+  def test_filedeps_one_target(self):
+    args = [
+      '--no-v1',
+      '--v2',
+      'filedeps',
+      'testprojects/tests/python/pants/dummies:passing_target',
+    ]
+    pants_run = self.run_pants(args)
+    self.assert_success(pants_run)
+
+    self.assertEqual("", pants_run.stderr_data)
+    self.assertEqual(pants_run.returncode, 0)
+
+    self.assertEqual(pants_run.stdout_data,
+      '''testprojects/tests/python/pants/dummies/BUILD
+testprojects/tests/python/pants/dummies/test_pass.py
+''')
+
+  def test_filedeps_one_target_with_deps(self):
```

Comment:
Let's maybe get rid of this test - it's covered by the test below, and integration tests take a while to run.

Reply:
Maybe I should get rid of the first two tests and leave only the last one because it covers previous cases?

Reply:
Works for me :)